src/main/java/dream/fcard/model/Deck.java

```diff
@@ -34,7 +32,7 @@
     /**
```

Comment:
if stats is not fully stable yet leave deckstats and statistics out of deck so that the rest of the members don't have to merge conflicts when they PR in also but if deckstats and statistics already work then its fine

Reply:
yep, using DeckStub for now like you suggested :)